python/ray/_private/usage/usage_lib.py

```diff
@@ -237,6 +238,37 @@ def _put_library_usage(library_usage: str):
             logger.debug(f"Failed to write a library usage to the home folder, {e}")
 
 
+def record_extra_usage_tag(key: str, value: str):
```

Comment:
Do we disallow `;`?

Reply:
We only disallow `;` for extra tags set via env var.